### PR TITLE
Fixes #1341 add label props and error props for input

### LINKED ISSUE #1341: Add label props for label input
Similar to `ListItem` where we can define specific props for title, rightTitle etc via the corresponding props, I'd like to propose the addition of `labelProps` for the `Input` component.

To be consistent, one could consider also adding an `InputProps` for the actual InputText but this is somewhat already (implicitly) available with the [attributes](https://github.com/react-native-training/react-native-elements/blob/d4da856a7f105c814cc162e507b48c49399cabef/src/input/Input.js#L98) props already 

I am going to submit a PR for that.

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
Adding labelProps and errorProps to Input component

**Comment by msevestre** [comment]
lol. Ok it's in this repo :) Will fix asap

**Comment by msevestre** [comment]
@iRoachie done

File: docs/input.md [hunk 1]
```diff
@@ -104,6 +106,16 @@ add styling to error message (optional)
 
 ---
 
+### `errorProps`
+
+provide all props from react-native Text component
```
Comment (iRoachie) [inline docs/input.md]: May want to be more descriptive here.
Like: `Allows custom props you can pass to the error text`

File: src/index.d.ts [hunk 2]
```diff
@@ -752,6 +752,11 @@ export interface InputProps extends TextInputProperties {
    */
   errorMessage?: string;
 
+  /**
+   * 	Props for label (optional)
```
Comment (iRoachie) [inline src/index.d.ts]: You have label here

Comment (msevestre) [inline src/index.d.ts]: ah crap. Sorry man

**Comment by iRoachie** [comment]
Ha awesome. Thanks for all the work back and forth.

**Pull request merged**